styles.css

```diff
@@ -709,14 +673,182 @@ hr {
   left: 35%;
 }
 
+/* ======Popup================================================================================== */
+
+.header-pop {
+  width: 18.438rem;
+  height: 40px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2.188rem;
+  line-height: 2.75rem;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 1rem;
+  margin-bottom: 39px;
+}
+
+.pop_description {
+  list-style: none;
+  height: 2.313rem;
+}
+
+ul.pop_description {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 5%;
+  margin-right: 16px;
+}
+
+.popup-header {
+  padding-top: 1px;
+}
+
+.popup_xclose {
+  position: absolute;
+  left: 72vw;
+  padding-top: 4px;
+  width: 15px;
+  height: 15px;
+  filter: opacity(0.5) drop-shadow(0 0 black);
+  top: 4vh;
+}
+
+.pop_canopy {
+  width: 3.438rem;
+  height: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 1rem;
+  line-height: 1rem;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-right: 12px;
+}
+
+#pop_counter {
+  width: 0.5rem;
+  height: 0.5rem;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.pop_back {
+  width: 6.125rem;
+  height: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.875rem;
+  line-height: 1rem;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.pop_image {
+  width: 18.438rem;
+  height: 20rem;
+  border-radius: 0.5rem;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 auto;
+  display: block;
+}
+
+.popdsk_image {
+  display: none;
+}
+
+.pop_more-info {
+  margin-bottom: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 1.5rem;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-left: 1rem;
+}
+
+.pop_languages {
+  width: 1.813rem;
+  height: 3.25rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.25rem;
+  line-height: 1rem;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  display: flex;
+  list-style: none;
+}
+
+.pop_lang {
+  margin: 0 0.625rem;
+  padding: 10px 6px 0 4px;
+  border-radius: 7px;
+  margin-top: 0.125rem;
+  background-color: #ebebff;
+  width: fit-content;
```

Comment:
- [ ] I would kindly suggest you use a CSS rule on you styling, what if you can use % for the width instead of px or rem? I think this will also help your website responsiveness, kindly fix this and ensure your website is responsive on both mobile and desktop popup

Reply:
Use % instead of  rem